### PR TITLE
Update swf module

### PR DESCRIPTION
**Pull request by radtriste**
I have 3 scripts in the builder image so that the user may reuse if needed
- create-app => does create the quarkus application with default extensions
- add-extension.sh to easily add new extensions
- build-app.sh to build the application with the given resources

That way the user will:
- default use the scaffold project and call the build-app.sh script
- can add extensions easily, either via the add-extension script or via the QUARKUS_EXTENSIONS env var and call the build-app script
- can recreate the scaffold project by deleting it and call the create-app script with own project groupid/artifactid/version

### CODE REVIEW

**Review by desmax74 — APPROVED** [review]

File: modules/kogito-swf-builder/added/add-extension.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+set -e
+
+script_dir_path="$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)"
+extensions="$1"
+
+# Call the configure-maven here
+source "${script_dir_path}/configure-maven.sh"
+configure
+
+set -x
```
Comment (spolti) [inline modules/kogito-swf-builder/added/add-extension.sh]: better to use debug env to control whether or not to enable the debug.

Comment (radtriste) [reply modules/kogito-swf-builder/added/add-extension.sh]: update

File: modules/kogito-swf-builder/added/add-extension.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -e
+
+script_dir_path="$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)"
+extensions="$1"
+
+# Call the configure-maven here
+source "${script_dir_path}/configure-maven.sh"
+configure
```
Comment (spolti) [inline modules/kogito-swf-builder/added/add-extension.sh:9]: does the builder image have a entrypoint?

Comment (radtriste) [reply modules/kogito-swf-builder/added/add-extension.sh:9]: nope. AFAIK you need to use it in a Dockerfile

**Review by spolti — APPROVED** [review]

**Pull request merged**